.isort.cfg

```diff
@@ -0,0 +1,12 @@
+[settings]
+profile=black
+
+sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_first_party=common,data_refresh,database,maintenance,oauth2,providers,retired
```

Comment:
It seems that because the `dags` directory is the airflow pipeline folder, isort is run with `dags` as the root, and I didn't find a way to dynamically list all subdirectories here 🤔